### DISCUSSION TITLE
Conda issue on collecting Package

### DISCUSSION THREAD

**Discussion by JoeLaFripouille** [facebookresearch/pytorch3d#1285] [Q&A]
It's impossible  for me to download the package via conda :

"Collecting package metadata (current_repodata.json): ...working... failed
Note: you may need to restart the kernel to use updated packages.

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/pytorch3d/win-64/current_repodata.json>
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
'https://conda.anaconda.org/pytorch3d/win-64'"

Do you have any solution ? 
thanks !

**Comment by bottler** [comment]
We have no conda packages for windows. If you need to install PyTorch3D on windows you need to build from source, which is not trivial. Look at some other issues for instructions.
